src/components/GoCookieConsent/GoCookieConsent.vue

```diff
@@ -7,10 +7,37 @@
     >
       <div class="cookie-consent__content">
         <slot name="content">
-          <p
-            class="text-white m-r-xs"
-            v-html="textContent"
-          />
+          <div>
+            <p
+              v-if="isVueComponent"
+              class="text-white m-r-xs"
+            >
+              <component :is="textContent" />
```

Comment:
`textContent` name for a potential component name is not really a good thing I think.